### DISCUSSION TITLE
How to stop creating payload.js files in static websites.

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#9963] [Questions (Nuxt 2)]
Hi, I have a sample static website for a movies website. I create it just for testing. now I have a problem. when I generate my website, Nuxt creates a folder and an HTML file for every movie like:
"dist\movies\tt0092593\index.html"
and render all movies data like title, release date, and etc. but also creates a payload.js file in:
"dist\_nuxt\static\1634719196\movies\tt0092593\payload.js"
and in this file, all of the movie data are available in JSON format. 

I have two problems:
1- data are duplicated: there is one series of movie info in HTML format and one in JSON format and I don't want a JSON version of it. I don't want the reactivity version of these data after rendering to HTML. I want to destroy it after rendering it to HTML. is there any other usage other than reactivity?
2- is my data safe? a simple crawler could crawl all of my website data in a pretty JSON format!

**Comment by siolfyr** [comment]
@Nik96i Any code you can show? Is this a page for an individual movie, using asyncData?
